apps/web/src/scenes/UserGalleryPage/UserGallery.tsx

```diff
@@ -92,7 +94,7 @@ function UserGallery({ queryRef }: Props) {
 
   return (
     <VStack gap={isMobile ? 12 : 24}>
-      <VStack gap={12}>
+      <VStack gap={user?.bio || !isMobile ? 12 : 0}>
```

Comment:
I think a better approach here  would be to have the `UsernameAndDescriptionHeader` just return null if it has no content. That way there's no extra gap and we can keep this number constant.

Reply:
big brain 🧠 . Updated the component!